### PR TITLE
Remove version admonitions from ShiftStack docs

### LINKED ISSUE #20786: ShiftStack version info is duplicated
### Which section(s) is the issue in?

From @codyhoag:

> I stumbled on [this PR](https://github.com/openshift/openshift-docs/pull/20603) today updating the OSP supported versions for OCP 4.3. It appears it was updated for OSP 13, 15, and 16.
> 
> When I was covering for you back in December, the versioning was brought up and it was requested to add the versioning info to the beginning of the assemblies for installing on OSP and installing on OSP with Kuryr. However, at that time, it was communicated to me that only versions 13 and 16 (not 15) were supported for OCP 4.3.

Example: https://docs.openshift.com/container-platform/4.3/installing/installing_openstack/installing-openstack-installer-custom.html

Look at the first _Important_ admonition.

### What needs fixing?

There's no need to include this info on these pages--it just creates a maintenance problem. I'll remove the top-of-assembly statements so that the sole source of truth is in the arch docs. 

https://docs.openshift.com/container-platform/4.3/architecture/architecture-installation.html#available-platforms_architecture-installation

### PR DESCRIPTION
**Pull request by maxwelldb**
Resolves #20786

### CODE REVIEW

**Comment by maxwelldb** [comment]
@kalexand-rh Can you approve/deny this? As the issue shows, the information I pulled is already described in the arch docs.

File: installing/installing_openstack/installing-openstack-installer-custom.adoc [hunk 1]
```diff
@@ -18,17 +18,6 @@ processes.
 
 * Have metadata service enabled in {rh-openstack}
 
-[IMPORTANT]
-====
-{product-title} 4.3 is supported for use with {rh-openstack} 13 and
-{rh-openstack} 16.
-
-The latest {product-title} release supports both the latest {rh-openstack} long
```
Comment (kalexand-rh) [inline installing/installing_openstack/installing-openstack-installer-custom.adoc]: There's more data in this note than in the arch guide, so I hesitate to just remove it. I suggest implementing one of these changes:

A. Update the arch guide to contain more nuance about the type of OSP releases that you're supporting. Amend the prereq about consulting the arch guide to point out that you need to care about the versions.

B. Make a module about supported OSP versions and include it in the assemblies, like [vSphere](https://docs.openshift.com/container-platform/4.2/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-infrastructure_installing-vsphere). Pull the details of supported OSP versions from the arch doc.

As long as the details are in only one place, I'm fine with either. I'd probably do B and swing it by Eric Rich.

**Comment by maxwelldb** [comment]
@kalexand-rh How 'bout that?

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Comment by maxwelldb** [comment]
Manual it is.

**Pull request merged**